src/metabase/lib/metadata/jvm.clj

```diff
@@ -347,10 +354,11 @@
 (defn- metadatas-for-table [metadata-type table-id]
   (case metadata-type
     :metadata/column
-    (t2/select :metadata/column
-               :table_id        table-id
-               :active          true
-               :visibility_type [:not-in #{"sensitive" "retired"}])
+    (into [] (map #(assoc % :ident (str (get *table-idents* table-id) "__" (:name %))))
```

Comment:
Why does this work differently from `t2/define-after-select :metadata/column`? (One of them adds `:ident` even if `(get *table-idents* table-id)` is missing, the other does not.)